app/routes/install.js

```diff
@@ -0,0 +1,7 @@
+import Route from '@ember/routing/route';
+
+export default Route.extend({
+    redirect() {
+        window.location = 'http://doc.crates.io/';
```

Comment:
Should this be `https`? Or protocol-relative, so just `//doc.crates.io`?

Reply:
Unfortunately, `https` doesn't work on CNAME'd gh-pages, yet. Making this HTTPS or relative would make a false alarm like this:

![image](https://user-images.githubusercontent.com/37169/30103952-0a7c9b0e-92a9-11e7-8dec-5c9ec7157e4c.png)